client/app/dedicated/server/rendezvous/dedicated-server-rendezvous.controller.js

```diff
@@ -0,0 +1,39 @@
+angular.module('App').controller('DedicatedServerRendezVousCtrl', class DedicatedServerRendezVousCtrl {
+  constructor($q, $state, $stateParams, $translate, Alerter, ovhUserPref) {
+    this.$q = $q;
+    this.$state = $state;
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.ovhUserPref = ovhUserPref;
+    this.Alerter = Alerter;
+    this.stopBother = false;
+    this.serversRendezVousToStopBother = [];
+  }
+
+  $onInit() {
+    this.ovhUserPref
```

Comment:
Prefer always returning promises
```suggestion
    return this.ovhUserPref
```